migrate/20250501_discount_code.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:discount_code) do
+      column :id, :uuid, primary_key: true
+      column :created_at, :timestamptz, null: false, default: Sequel::CURRENT_TIMESTAMP
+      column :discount_code, :text, null: false, unique: true
+      column :credit_amount, :numeric, null: false
+      column :expires_at, :timestamptz, null: false
+    end
+
+    create_table(:project_discount_code) do
+      column :id, :uuid, primary_key: true
+      column :created_at, :timestamptz, null: false, default: Sequel::CURRENT_TIMESTAMP
+      foreign_key :project_id, :project, type: :uuid, null: false
```

Comment:
should we make the combination of `project_id` and `discount_code_id` unique, so that a single project cannot apply the same discount multiple times?

Reply:
Example of a uniqueness constraint: https://github.com/ubicloud/ubicloud/blob/e7d6e6188d22969c93e22e4cc97d1dbbe21252ff/migrate/20230620_storage.rb#L12